driver/src/main/java/org/neo4j/driver/internal/async/ChannelConnectedListener.java

```diff
@@ -47,21 +48,22 @@ public ChannelConnectedListener( BoltServerAddress address, ChannelPipelineBuild
         this.pipelineBuilder = pipelineBuilder;
         this.handshakeCompletedPromise = handshakeCompletedPromise;
         this.logging = logging;
-        this.log = logging.getLog( getClass().getSimpleName() );
     }
 
     @Override
     public void operationComplete( ChannelFuture future )
     {
         Channel channel = future.channel();
+        Logger log = new ChannelActivityLogger( channel, logging, getClass() );
 
         if ( future.isSuccess() )
         {
-            log.trace( "Channel %s connected, running bolt handshake", channel );
+            log.trace( "Channel %s connected, initiating bolt handshake", channel );
```

Comment:
`ChannelActivityLogger` will also prefix messages with channel id, so it will appear twice in the message. Maybe better keep using regular logger?